deploy/kubernetes/webhook-example/README.md

```diff
@@ -0,0 +1,67 @@
+# How to deploy the webhook
+
+The webhook server is provided as an image which can be built from this repository. It can be deployed anywhere, as long as the api server is able to reach it over HTTPS. It is recommended to deploy the webhook server in the cluster as snapshotting is latency sensitive. A `ValidatingWebhookConfiguration` object is needed to configure the api server to contact the webhook server. Please see the [documentation](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/) for more details. The webhook server code is adapted from the [webhook server](https://github.com/kubernetes/kubernetes/tree/v1.18.6/test/images/agnhost/webhook) used in the kubernetes/kubernetes end to end testing code.
```

Comment:
should include a prerequisites/dependencies section in this README to include:
1. k8s version
2. admission controller is enabled etc

Reply:
We should also add a brief description of the webhook and add a link to this doc in the top level README: https://github.com/kubernetes-csi/external-snapshotter/blob/master/README.md